.happy/terraform/modules/sfn/main.tf

```diff
@@ -162,13 +162,26 @@ resource "aws_sfn_state_machine" "state_machine" {
         "Type": "Task",
         "InputPath": "$",
         "Resource": "${var.lambda_success_handler}",
-        "End": true
+        "End": true,
+        "Retry": [ {
+            "ErrorEquals": ["Lambda.AWSLambdaException"],
+            "IntervalSeconds": 1,
+            "MaxAttempts": 3,
+            "BackoffRate": 2.0
+          }]
+        }
```

Comment:
I think this might be one curly brace too many, HandleErrors below should still be inside 'States'

Reply:
If you update an rdev stack with this it should validate this code for you

Reply:
@ebezzi it does work